src/main/java/org/opensrp/web/rest/PractitionerRoleResource.java

```diff
@@ -107,6 +111,32 @@ public void setPractitionerRoleService(PractitionerRoleService practitionerRoleS
         }
     }
 
+
+    @RequestMapping(value = "/addOrUpdateMany", method = RequestMethod.POST, consumes = { MediaType.APPLICATION_JSON_VALUE,
+            MediaType.TEXT_PLAIN_VALUE })
+    public ResponseEntity<String> createOrUpdateMany(@RequestBody String entity) {
```

Comment:
This method should be called `save` or `batchSave` for consistency with the other APIs